### PR TITLE
modal for add emoji alias

### PR DESCRIPTION
**Pull request by songgao**

### CODE REVIEW

File: shared/util/emoji.tsx [hunk 1]
```diff
@@ -12,20 +13,13 @@ export type EmojiData = {
...
-      }
-    : {
-        category: category ?? '',
-        name: null,
-        short_name: emoji.alias,
-        short_names: [emoji.alias],
-        source: emoji.source.httpsrv,
-        unified: '',
-      }
+  return {
```
Comment (hyperobject) [inline shared/util/emoji.tsx:16]: picker and autocomplete don't handle aliases well but that's probably another ticket

File: shared/teams/team/rows/emoji-row/add.tsx [hunk 2]
```diff
@@ -25,7 +25,17 @@ const AddEmoji = ({teamID, convID, filter, setFilter}: OwnProps) => {
         ],
       })
     )
-  const onAddAlias = () => {}
+  const onAddAlias = () =>
```
Comment (hyperobject) [inline shared/teams/team/rows/emoji-row/add.tsx:28]: would be good to hook this up to the "add alias"/"edit alias" menu buttons in `emoji-row/item.tsx`

File: shared/util/emoji.tsx (hunk 1 above)
Comment (songgao) [reply shared/util/emoji.tsx:16]: I think with this change alias now works in the picker. I'll look into autocomplete

**Comment by songgao** [comment]
@hyperobject Please take another look; thanks!

New changes:

* hooked up the alias modal to the item popup
* I don't think we are doing "Edit Alias" so I just removed it
* added the call for removing emoji
* reload user emojis after adding an emoji/alias, or removing an emoji
* fixed alias in autocompleter

I noticed the team sectionlist flashes when it reloads and I'm still looking into that.

**Comment by songgao** [comment]
Alias to stock emoji doesn't render correctly on mobile yet. It's cut-off on top. Will visit again after https://github.com/keybase/client/pull/23533 is in.

**Pull request merged**